File: 2022/day/go_template/part2/part2.go
package main

import (
	"os"
	"regexp"
	"strings"
)

func main() {
	dat, _ := os.ReadFile("data/input.txt")

	lines := strings.Split(string(dat), "\n")

	for _, line := range lines {
		re := regexp.MustCompile(``)
		re.FindStringSubmatch(line)
	}

}

File: 2022/day/rust_template/src/bin/part2.rs
use std::fs;

use regex::Regex;

fn main() {
    let contents = fs::read_to_string("data/input.txt").expect("");

    let lines: Vec<&str> = contents.split('\n').collect();

    for line in lines {
        let re = Regex::new(r"move ([0-9]+) from ([0-9]+) to ([0-9]+)").unwrap();
        let _ = re.captures(line).unwrap();
    }
}
